api/src/controllers/connected-user-log.js

```diff
@@ -0,0 +1,42 @@
+const auth = require('../auth');
+const serverUtils = require('../server-utils');
+const connectedUserLogService = require('../services/connected-user-log');
+
+module.exports = {
+  log: (req) => {
+    return auth
+      .getUserCtx(req)
+      .then((userCtx) => {
+        const log = {
+          user: userCtx.name,
+          timestamp: new Date().getTime()
+        };
+        connectedUserLogService.save(log);
+      });
+  },
+  get: (req, res) => {
```

Comment:
I think this should be added into the existing monitoring API rather than a whole new API. If the logging is done as a middleware and the `get` is merged into the monitoring API then this whole controller goes away...